README.md

```diff
@@ -277,10 +277,22 @@ A comma-separated list of the packages to be installed on the system specified v
 
 #### Users
 
+##### ENABLE_USER=true
+
+Makes Pieman create a regular user `USER_NAME` with the password `USER_PASSWORD`.
+
 ##### PASSWORD="secret"
 
 Allows specifying the root password. It's **HIGHLY RECOMMENDED** to change the default root password.
 
+##### USER_NAME="cusdeb"
+
+Allows specifying the name for the only non-root user. See `ENABLE_USER`. It's ignored if `ENABLE_USER` is set to `false`.
```

Comment:
Allows specifying username (non-root user) if `ENABLE_USER` is set to `true`. Ignored if `ENABLE_USER` is `false`.